src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable quote-props */
+/* eslint-disable no-console */
+import classNames from 'classnames';
+import {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+import { Todo } from '../../types/Todo';
+
+// eslint-disable-next-line import/no-cycle
+import {
+  LoadingContext,
+  TodoUpdateContext,
+} from '../../TodosContext/TodosContext';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.